apps/ban-core-writer/src/index.ts

```diff
@@ -160,9 +138,9 @@ const getBanObjectsFromBalRows = (rows: any[]) => {
         districtID: row.id_ban_commune,
         mainCommonToponym: mainToponymes[row.id_ban_toponyme] || {},
         districts: districts[row.id_ban_commune] || {},
-        labels: getterLabels('lieudit_complement_nom') || [],
+        labels: row.ban_enrich_beautified_labels_lieudit_complement_nom || [],
```

Comment:
`ban_enrich_beautified_labels_lieudit_complement_nom` devrait toujours etre un tableau.

```suggestion
        labels: row.ban_enrich_beautified_labels_lieudit_complement_nom,
```